source/IOSResolver/src/IOSResolver.cs

```diff
@@ -1553,19 +1486,6 @@ public enum CocoapodsIntegrationMethod {
         project.ReadFromString(File.ReadAllText(pbxprojPath));
         string target = project.TargetGuidByName(TARGET_NAME);
         project.SetBuildProperty(target, "CLANG_ENABLE_MODULES", "YES");
-        project.AddBuildProperty(target, "OTHER_LDFLAGS", "$(inherited)");
-        project.AddBuildProperty(target, "OTHER_CFLAGS", "$(inherited)");
-        project.AddBuildProperty(target, "USER_HEADER_SEARCH_PATHS",
-                                 "$(inherited)");
-        project.AddBuildProperty(target, "USER_HEADER_SEARCH_PATHS",
-                                 "$(PROJECT_DIR)/" + PODS_DIR + "/Headers/Public");
-        project.AddBuildProperty(target, "HEADER_SEARCH_PATHS",
-                                 "$(PROJECT_DIR)/" + PODS_DIR + "/Headers/Private");
-        project.AddBuildProperty(target, "FRAMEWORK_SEARCH_PATHS",
-                                 "$(inherited)");
-        project.AddBuildProperty(target, "FRAMEWORK_SEARCH_PATHS",
-                                 "$(PROJECT_DIR)/Frameworks");
-        project.AddBuildProperty(target, "LIBRARY_SEARCH_PATHS", "$(inherited)");
         project.AddBuildProperty(target, "OTHER_LDFLAGS", "-ObjC");
```

Comment:
Wasn't sure about these two.  If GTMSessionFetcher has a podspec / xcconfig that sets these, then the pods integration will take care of them, and these lines can also be removed.